docker-compose-local.yml

```diff
@@ -3,10 +3,8 @@ services:
   iroha0:
     build: .
     image: iroha0:debug
-    volumes:
-      - './configs/peer:/config'
-      - './:/root/soramitsu/iroha'
     environment:
+      KURA_BLOCK_STORE_PATH: /chain
```

Comment:
Double check if this needs to be quoted. AFAIK, serde requires it to be double quoted.

Reply:
It works